### PR TITLE
Leaf722 enforce piv card sign in

### PR DESCRIPTION
**Pull request by jeanmarcablevets**
New branch that excludes unwanted things from `sensitive_indicators`.

### CODE REVIEW

File: LEAF_Request_Portal/auth_domain/index.php [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 $login = new Login($db_phonebook, $db);
 
-if (isset($_SERVER['REMOTE_USER']))
+if (isset($_SERVER['REMOTE_USER']) && (!isset(Orgchart\Config::$leafSecure) || Config::$leafSecure == false))
```
Comment (jeremyrl) [inline LEAF_Request_Portal/auth_domain/index.php]: Remove `Orgchart\` here

File: LEAF_Request_Portal/login/index.php [hunk 2]
```diff
@@ -60,10 +60,13 @@ function getBaseDir()
...
-    <a href="<?php echo '//' . $_SERVER['SERVER_NAME'] . getBaseDir() . 'auth_domain/?' . htmlentities($_SERVER['QUERY_STRING'], ENT_QUOTES | ENT_HTML5, 'UTF-8'); ?>" style="text-decoration: none"><div class="buttonNorm" style="text-align: center">Login with <b>Username and Password</b></div></a>
+        <a href="<?php echo '//' . $_SERVER['SERVER_NAME'] . ':444' . getBaseDir() . 'auth_token/?' . htmlentities($_SERVER['QUERY_STRING'], ENT_QUOTES | ENT_HTML5, 'UTF-8'); ?>" style="text-decoration: none"><div class="buttonNorm" style="text-align: center">Login with <b>PIV card</b><img src="../../libs/dynicons/?img=contact-new.svg&amp;w=32" style="padding-left: 8px" alt="Icon for PIV card" title="Icon for PIV card" /></div></a><br />
+        <?php if (!isset(Orgchart\Config::$leafSecure) || Config::$leafSecure == false)
```
Comment (jeremyrl) [inline LEAF_Request_Portal/login/index.php]: Remove `Orgchart\` here

File: LEAF_Request_Portal/auth_domain/index.php (hunk 1 above)
Comment (jeanmarcablevets) [reply LEAF_Request_Portal/auth_domain/index.php]: Addressed in commit `d72`

File: LEAF_Request_Portal/login/index.php (hunk 2 above)
Comment (jeanmarcablevets) [reply LEAF_Request_Portal/login/index.php]: Addressed in commit `d72`

**Pull request merged**